src/deepsparse/yolo/pipelines.py

```diff
@@ -197,9 +203,28 @@ def process_inputs(
             iou_thres=inputs.iou_thres,
             conf_thres=inputs.conf_thres,
             multi_label=inputs.multi_label,
+            original_image_shapes=original_image_shapes,
         )
         return [image_batch], postprocessing_kwargs
 
+    def _scale_boxes(
+        self, boxes: numpy.ndarray, original_image_shape: Optional[Tuple[int, ...]]
+    ):
+        if not original_image_shape:
+            return boxes
+
+        scale = numpy.flipud(
+            numpy.divide(
+                numpy.asarray(original_image_shape), numpy.asarray(self.image_size)
+            )
+        )
+
+        # broadcast scales to rescale bounding boxes in one numpy operation
+        scale = numpy.concatenate([scale, scale])
+
+        boxes = numpy.multiply(boxes, scale)
+        return boxes
+
     def _preprocess_image(self, image) -> numpy.ndarray:
```

Comment:
okay I missed this :)

```suggestion
    def _preprocess_image(self, image) -> Tuple[numpy.ndarray, Tuple[int, int]]:
```